### PR TITLE
Add hint about GraphQL Usage

### PR DESCRIPTION
**Pull request by tfrauenstein**
solves https://github.com/zalando/restful-api-guidelines/issues/514

### CODE REVIEW

**Comment by tkrop** [comment]
:+1:

File: chapters/general-guidelines.adoc [hunk 1]
```diff
@@ -43,6 +43,15 @@ existing APIs the https://editor.swagger.io/[Swagger Editor] or our
 https://apis.zalando.net[API Portal] may be a good starting point.
 
 
+*Hint:* We do not yet provide analog guidelines for https://graphql.org/[GraphQL].
```
Comment (whiskeysierra) [inline chapters/general-guidelines.adoc]:
```suggestion
*Hint:* We do not yet provide guidelines for https://graphql.org/[GraphQL]. 
```

Either we remove it or use it as a noun (in which case *analogue* might be more correct, not sure).

**Comment by tkrop** [comment]
:+1:

**Comment by whiskeysierra** [comment]
:+1:

**Pull request merged**